client/packages/coldchain/src/Equipment/DetailView/Tabs/StatusLogs.tsx

```diff
@@ -7,7 +7,7 @@ import {
   UserCircleIcon,
 } from '@openmsupply-client/common';
 import { useFormatDateTime, useIntlUtils, useTranslation } from '@common/intl';
-import { AssetLogFragment, useAssets } from '../../api';
```

Comment:
i've used a different fragment for accessing assetLogs within cold chain, because cold chain asset log fragment will also include document and log reasons